profiles/lib/way_handlers.lua

```diff
@@ -429,6 +429,66 @@ function WayHandlers.parse_maxspeed(source,profile)
   return n
 end
 
+function WayHandlers.parse_length_unit(source)
+  if not source then
+    return
+  end
+  local n = tonumber(source:match("%d+%.?%d*"))
+  if n then
+    inches = source:match("'.*")
+    if inches then -- Imperial unit to metric
+      n = n * 12
+      local m = tonumber(inches:match("%d+"))
+      if m then
+        n = n + m
+      end
+      n = n * 2.54 / 100
+    end
+    return n
+  end
+end
```

Comment:
We did something like this: https://github.com/vng/osrm-backend/blob/trucks/profiles/lib/measure.lua

Reply:
Nice. I planned to do a lib like this to read units.